neard/src/main.rs

```diff
@@ -78,6 +79,7 @@ fn main() {
                 .help("Directory for config and data (default \"~/.near\")")
                 .takes_value(true),
         )
+        .arg(Arg::with_name("no-ansi").long("no-ansi").takes_value(false).help("Disable ansi charset and colored logging."))
```

Comment:
I would never guessed that this option is about disabling color output.

Use `"color"` name (it should create the `no-color` counterpart automatically).

Reply:
I didn't dive into the code, but based on help string:

> Disable ansi charset and colored logging.

Option named `color` also doesn't reflect what help description is saying, so option named `ansi` sounds more appropriate.

Reply:
I added color first, but I changed it to ansi, for same reason @kucharskim has mentioned. Color is misleading, as it is not only color but any ansi specific chars. Also you cannot just disable color unfortunatelly.

Reply:
@chefsale which ansi characters do we use except the color control group?

Reply:
Most likely none, but for some reason I think we use up arrow possibly, but theoratically somebody could add it. I mean personally I don't think using --no-color is bad, but in case we ever add or have some ansi chars it won't work.

Reply:
Also one thing which I'm currently trying to find is the counterpart thing: `--color/--no-color`. I cannot really find anywhere that it actually does create a counterpart and how to parse it :confused: Do we have somewhere and example in the code where this works, as I'm not able to figure it out :smile:

Reply:
I think it makes sense to have color on by default and have something like `--no-color` or `--no-color-log` that disables it

Reply:
Hmm, I was wrong about the `--no-*` feature, you still need to implement it separately. I agree that `--no-color-log` is even better. The arrow is a unicode character, not an ansi control sequence

Reply:
Added it as --no-color flag.